core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -316,21 +318,25 @@ class ReassignPartitionsCommand(zkUtils: ZkUtils, proposedAssignment: Map[TopicA
   }
 
   def maybeLimit(throttle: Long) {
+    // Limit the throttle on currently moving replicas. Note that this command can use used to alter the throttle,
+    // but it may not alter all limits originally set, if some of the brokers have completed their rebalance.
     if (throttle >= 0) {
       val existingBrokers = existingAssignment().values.flatten.toSeq
       val proposedBrokers = proposedAssignment.values.flatten.toSeq
       val brokers = (existingBrokers ++ proposedBrokers).distinct
 
       for (id <- brokers) {
         val configs = AdminUtils.fetchEntityConfig(zkUtils, ConfigType.Broker, id.toString)
-        configs.put(DynamicConfig.Broker.ThrottledReplicationRateLimitProp, throttle.toString)
+        configs.put(DynamicConfig.Broker.ThrottledLeaderReplicationRateProp, throttle.toString)
+        configs.put(DynamicConfig.Broker.ThrottledFollowerReplicationRateProp, throttle.toString)
```

Comment:
It seems it makes more sense to take two throttle limits from the command line,  one for the leader and one for the follower?